docs/core/additional-tools/wcf-web-service-reference-guide.md

```diff
@@ -0,0 +1,78 @@
+﻿--- 
+title: Microsoft WCF Web Service Reference Provider Tool
+description: An overview of the Microsoft WCF Web Service Reference Provider Tool that adds functionality for .NET Core and ASP.NET Core projects, similar to Add Service Reference for .NET Framework projects.
+author: mlacouture
+manager: wpickett
+ms.author: johalex
+ms.date: 01/12/2018
+ms.topic: article
+ms.prod: .net-core
+ms.custom: mvc
+---
+# Microsoft WCF Web Service Reference Provider Tool
+
+Over the years, many Visual Studio developers have enjoyed the productivity that the [Add Service Reference](https://docs.microsoft.com/visualstudio/data-tools/how-to-add-update-or-remove-a-wcf-data-service-reference) tool provided when their .NET Framework projects needed to access web services. The **WCF Web Service Reference** tool is a Visual Studio connected service extension that provides an experience like the Add Service Reference functionality for .NET Core and ASP.NET Core projects. This tool retrieves metadata from a web service in the current solution, on a network location, or from a WSDL file, and generates a .NET Core compatible source file containing Windows Communication Foundation (WCF) client proxy code that you can use to access the web service.
+
+> [!IMPORTANT]
+> You should only reference services from a trusted source. Adding references from an untrusted source may compromise security. 
+
+## How to use the extension
+
+> [!NOTE]
+> The **WCF Web Service Reference** option is applicable to projects created using the following project templates:
+> * **Visual C#** > **.NET Core**  
+> * **Visual C#** > **.NET Standard**
+> * **Visual C#** > **Web** > **ASP.NET Core Web Application**
+
+Using the **ASP.NET Core Web Application** project template as an example, this article walks you through adding a WCF service reference to the project:
+
+1. In Solution Explorer, double-click the **Connected Services** node of the project (for a .NET Core or .NET Standard project this option is available when you right click on the **Dependencies** node of the project in Solution Explorer). 
+
+The **Connected Services** page appears as shown in the following image:
+
+![Connected Services tab](./media/wcf-web-service-reference-guide/WCFCS-ConnectedServicesPage.png)
```

Comment:
images are not rendering. they are still not inside a media folder. also here the image name is still using capital letters, it should be lowercase.

also alt text should be more descriptive,